src/controllers/users_controller.ts

```diff
@@ -1,30 +1,31 @@
-const User = require('../models/users_model');
-const token = require('../utilities/token');
-const bcrypt = require("bcrypt");
+import bcrypt from 'bcrypt';
+import { Request, Response } from "express";
+import * as token from '../utilities/token';
+import User, { UserDocument } from '../models/users_model';
 
 
-const getAllUsers = async (req, res) => {
+export const getAllUsers = async (req: Request, res: Response): Promise<any> => {
     try {
-      const users = await User.find();
-      return res.json(users);
+      const users : UserDocument[] | null= await User.find();
+      return res.status(200).json(users);
```

Comment:
```suggestion
      return res.json(users);
```
`200` is the default